exercises/largest-series-product/.meta/src/reference/java/LargestSeriesProductCalculator.java

```diff
@@ -32,15 +32,15 @@ public long calculateLargestProductForSeriesLength(final int seriesLength) throw
         }
     }
 
-    private void validateStringToSearch() throws IllegalArgumentException {
+    void validateStringToSearch() throws IllegalArgumentException {
         if (stringToSearch == null) {
             throw new IllegalArgumentException("String to search must be non-null.");
         } else if (!stringToSearch.chars().allMatch(Character::isDigit)) {
             throw new IllegalArgumentException("String to search may only contain digits.");
         }
     }
 
-    private long computeProductOfSeries(final int startIndex, final int seriesLength) {
```

Comment:
I think that these `private` methods can stay private. Do you agree? :slightly_smiling_face: